proto/protos/sst.proto

```diff
@@ -15,6 +15,7 @@ message SstBloomFilter {
   repeated RowGroupFilter row_group_filters = 1;
 }
 
+/// Used by ssts encoded by parquet, incuding columar&hybrid storage formats.
 message SstMetaData {
```

Comment:
How about add one more level abstration:
```
message SstMetaData {
  oneof MetaInfo {
    ParquetMetaData data = 1;
  }
}

message ParquetMetaData {
  // Min key in the sst
  bytes min_key = 1;
  // Max key in the sst
  bytes max_key = 2;
  // Max sequence number in the sst
  uint64 max_sequence = 3;
  // The time range of the sst
  common.TimeRange time_range = 4;
  common.TableSchema schema = 5;
  uint64 size = 6;
  uint64 row_num = 7;
  analytic_common.StorageFormatOptions storage_format_opts = 8;
  SstBloomFilter bloom_filter = 9;
}

```

Reply:
It sounds great!